README.md

```diff
@@ -174,6 +174,7 @@ Example values:
 | mindfulness     | 1800 <br/>**Notes**: only available on iOS |
 | nutrition      | { item: "cheese", meal_type: "lunch", brand_name: "McDonald's", nutrients: { nutrition.fat.saturated: 11.5, nutrition.calories: 233.1 } } <br/>**Note**: the `brand_name` property is only available on iOS |
 | nutrition.X    | 12.4                              |
+| sleep       | 2 <br />**Notes**: recognized sleep stages and their mappings in Google Fit can be found [here](https://developers.google.com/android/reference/com/google/android/gms/fitness/data/SleepStages#constant-summary) |
```

Comment:
specify the categories here:

sleep, sleep.awake, sleep.outOfBed, sleep.deep, sleep.light, sleep.rem